miner/worker.go

```diff
@@ -1019,11 +1019,11 @@ func (w *worker) commitTransactions(txs types.OrderedTransactionSet, coinbase co
 		l2CommitTxsTimer.Update(time.Since(t0))
 	}(time.Now())
 
-	var circuitCapacityReached bool
+	var ccOrBtReached bool // CircuitCapacity or BlockTime Reached
```

Comment:
feels a little strange with `ccOrBtReached`, maybe `circuitCapacityOrBlockTimeReached` is clear